admin_tests/base/test_utils.py

```diff
@@ -108,3 +111,41 @@ def test_rules_to_subjects(self):
         returned_subjects = rules_to_subjects(rules)
 
         self.assertItemsEqual(subject_queryset_ideal, returned_subjects)
+
+class TestNodeChanges(AdminTestCase):
+    def setUp(self):
+        super(TestNodeChanges, self).setUp()
+        self.registration = RegistrationFactory()
+        self.user = UserFactory()
+        self.user.is_staff = True
+        self.user.groups.add(Group.objects.get(name='osf_admin'))
+        self.user.save()
+
+        self.date1 = self.registration.registered_date + datetime.timedelta(days=365)
```

Comment:
These could be more descriptive to clarify the expected outcome -- date_valid, date_long_invalid,  etc